commerce_coordinator/settings/local.py

```diff
@@ -4,6 +4,7 @@
 
 INSTALLED_APPS += (
     'commerce_coordinator.apps.demo_lms.apps.LmsConfig',
+    # 'commerce_coordinator.apps.orders.apps.OrdersConfig',
```

Comment:
Should this be commented out like this?

Reply:
Oops sorry- I wasn't sure if I'd need that, and it looks like I don't- removed!